packages/kobotoolbox/package.json

```diff
@@ -25,7 +25,7 @@
     "configuration-schema.json"
   ],
   "dependencies": {
-    "@openfn/language-common": "workspace:^1.8.1"
+    "@openfn/language-common": "^1.8.1"
```

Comment:
This diff isn't necessary - we're going to trigger a patch release for no change to the final package. There's quite a few like this. I think `workspace:^...` dependencies are perfectly safe.